scripts/test-infra/test-e2e-presubmit.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/sh
+set -o errexit
```

Comment:
nit/suggestion: I think we use `bash` in the other test-infra scripts so we could follow that. Also, I think `pipefail` would be a good safety measure (for possible future changes, I don't think it's critical with this code)

```suggestion
#!/bin/bash -e
set -o pipefail
```

Reply:
thanks, fixed